haystack/components/agents/agent.py

```diff
@@ -218,6 +224,36 @@ def _create_agent_span(self) -> Any:
             },
         )
 
+    def _stream_tool_call(self, message: ChatMessage, streaming_callback: Optional[StreamingCallbackT] = None) -> None:
+        """
+        Stream information about a tool call.
```

Comment:
Just to be complete I'd expand on this docstring to explain that we send the full result in one streaming chunk. So again to emphasize it's not true streaming but it still uses the steraming_callback to send the info.